clients/include/rocblas_math.hpp

```diff
@@ -20,7 +20,14 @@ inline __host__ rocblas_half float_to_half(float val)
 }
 
 // Helper routine to convert halfs into their floats equivalent; uses F16C instructions
-inline __host__ float half_to_float(rocblas_half val)
+template <typename T>
+inline __host__ float half_to_float(T val)
+{
+    throw rocblas_status_internal_error;
```

Comment:
We do not throw exceptions in test code right now.

Maybe this was accidentally left in during debugging.

Reply:
understand. For this function, I will return val directly for non-half type. Is that ok??